src/ast/nodes/ArrowFunctionExpression.ts

```diff
@@ -75,11 +75,15 @@ export default class ArrowFunctionExpression extends FunctionBase {
 		return isIIFE || super.onlyFunctionCallUsed();
 	}
 
-	include(context: InclusionContext, includeChildrenRecursively: IncludeChildren): void {
-		super.include(context, includeChildrenRecursively);
+	includePath(
+		path: ObjectPath,
+		context: InclusionContext,
+		includeChildrenRecursively: IncludeChildren
+	): void {
+		super.includePath(path, context, includeChildrenRecursively);
 		for (const parameter of this.params) {
 			if (!(parameter instanceof Identifier)) {
-				parameter.include(context, includeChildrenRecursively);
+				parameter.includePath(path, context, includeChildrenRecursively);
```

Comment:
That seems wrong to me. So let's say we have

```
const foo = (a) => {};
console.log(foo.bar);
```

Then that should include the path `['bar']` of 'foo', but this path has no relevance for the parameters. We could include `UNKNOWN_PATH` if we want to be safe.